src/error.cpp

```diff
@@ -0,0 +1,38 @@
+#include "error.h"
+#include <string>
+
+namespace lib
```

Comment:
Let's call the namespace `RadProto` and put everything in it, including `Packet`, `Server`, attributes and dictionaries.